### PR TITLE
Disable debug log messages

### PR DESCRIPTION
**Pull request by pankajgoyal2**

### CODE REVIEW

**Comment by agunde406** [comment]
There are two changes in this commit. Changing cargo build from debug to release mode, and removing the logging debug option. They should be in separate commits and a reason should be provided for each.

File: src/main.rs [hunk 1]
```diff
@@ -128,8 +128,7 @@ fn parse_args() -> RaftCliArgs {
     let log_level = match matches.occurrences_of("verbose") {
         0 => log::LevelFilter::Warn,
         1 => log::LevelFilter::Info,
-        2 => log::LevelFilter::Debug,
```
Comment (peterschwarz) [inline src/main.rs]: This shouldn't be removed, but an evaluation of what messages are being logged as "debug" and what are logged as "trace".

File: adhoc/node.yaml [hunk 2]
```diff
@@ -103,7 +103,7 @@ services:
     volumes:
       - ${SAWOOTH_RAFT:-..}:/project/sawtooth-raft
     working_dir: /project/sawtooth-raft/
-    command: ./target/debug/raft-engine --connect tcp://validator:5050 -v
+    command: ./target/release/raft-engine --connect tcp://validator:5050 -v
```
Comment (peterschwarz) [inline adhoc/node.yaml]: This change should be made in a separate commit (and possibly a separate PR, as it is less controversial than the logging change.
